src/core/Button/Button.tsx

```diff
@@ -10,12 +10,12 @@ type ButtonVariant =
   | 'inverted'
   | 'secondary'
   | 'secondary-noborder'
```

Comment:
I think this might actually be bad practice / not according to our conventions to have the variant name this way. Should it be in lowerCamelCase?

Reply:
Or do we have similar variant names somewhere else?

Reply:
That's the original name from 2 years ago. I don't think we have kebab-case names elsewhere. Should probably update this one if we are breaking things anyway.

Reply:
As discussed elsewhere, I renamed the variant to secondaryNoBorder to put it in line with the general naming policy.